spec/components/sdg/goals/targets_component_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+describe SDG::Goals::TargetsComponent, type: :component do
+  let(:goal) { SDG::Goal[1] }
+  let(:component) { SDG::Goals::TargetsComponent.new(goal) }
+
+  before do
+    Setting["feature.sdg"] = true
+  end
+
+  it "does not render when the feature is disabled" do
+    Setting["feature.sdg"] = false
+
+    render_inline component
+
+    expect(page).not_to have_css ".targets"
+    expect(page).not_to have_css "#target_tabs"
+    expect(page).not_to have_css ".tabs-content"
+  end
+
+  it "renders tabs panel" do
+    render_inline component
+
+    expect(page).to have_css ".targets"
```

Comment:
You could avoid these expect and use de `and` method.
like 
```
expect(page).to have_css ("".targets").and(have_content("Targets")).and(have_content(""Local targets"))
```

Reply:
Thanks for the comment! However, since the "one expectation per line" pattern is used in the rest of the code, here we're using it for consistency.